build/buildConfig.js

```diff
@@ -38,7 +39,9 @@ module.exports = (bundles, themeEntries, paths, extractThemesPlugin, prod, publi
     }, bundles, themeEntries),
     mode: prod ? "production" : "development",
     optimization: {
-        minimize: false
+        minimize: false,
+        moduleIds: "named",
```

Comment:
Without named modules you cannot load code dynamically, because libraries are imported using a number, instead of the real name of a library. This ensures module ids are always the same, and do not change every compile you do.